util/tar.go

```diff
@@ -4,103 +4,123 @@ import (
 	"archive/tar"
 	"errors"
 	"io"
-	"log"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 )
 
-func Unpack(tarFile string) ([]string, error) {
+const TmpDirPrefix = "govcd"
+
+//extract files to system tmp dir with name govcd+random number. Created folder with files isn't deleted.
```

Comment:
What is the output of this function?  Also 'extract' should be capitalized.

Reply:
Done.